### DISCUSSION TITLE
Credentials???

### DISCUSSION THREAD

**Discussion by Rigolino1** [spencerwooo/onedrive-vercel-index#826] [General]
where obtain the:

clientId
obfuscatedClientSecret
authApi
driveApi

**Comment by lightumcc** [comment]
[Azure Active Directory](https://portal.azure.com/#view/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/~/RegisteredApps)
Follow the [instruction](https://ovi.swo.moe/docs/getting-started) to create your own app

**Reply by Rigolino1** [reply]
thx, i ilready deploy with your help <3

**Reply by Rigolino1** [reply]
Bro can you help me with another ask? Pleaseeee
